projects/story-decorator/src/StoryDecorator.ts

```diff
@@ -46,8 +46,10 @@ const urlParams = new URLSearchParams(queryString);
 
 export let dir: 'ltr' | 'rtl' =
     (urlParams.get('sp_dir') as 'ltr' | 'rtl') || 'ltr';
-export let theme: ThemeVariant =
-    (urlParams.get('sp_theme') as ThemeVariant) || 'spectrum';
+export let theme: SystemVariant =
+    (urlParams.get('sp_theme') as SystemVariant) || 'spectrum';
+export let system: SystemVariant =
+    (urlParams.get('sp_system') as SystemVariant) || 'spectrum'; // Not sure of how this export is used, do we also need to introduce an `sp_system`?
```

Comment:
Do we need to resolve the use of this export? I think this was related to Storybook but if we don't need to do anything different here, let's remove this comment?

Reply:
removed. Thanks for pointing it out